rwiki/rwiki-impl/impl/src/java/uk/ac/cam/caret/sakai/rwiki/component/macros/IndexMacro.java

```diff
@@ -240,14 +265,17 @@ private void emitGoDown(Writer writer, char[] chars, int start, int end)
 	}
 
 	private void emitListItem(Writer writer, PageLinkRenderer plr,
-			char[] chars, int start) throws IOException
+			char[] chars, int start, String groupsString) throws IOException
 	{
 
 		writer.write("<li>"); //$NON-NLS-1$
 		StringBuffer sb = new StringBuffer(chars.length * 3);
 		plr.appendLink(sb, new String(chars), new String(chars, start,
 				chars.length - start), null, true);
 		writer.write(sb.toString());
+		if (groupsString != null && !groupsString.equals("")) {
```

Comment:
StringUtils.isNotBlank(groupsString)

Reply:
what library?

Reply:
StringUtils, commons-lang3, I think it's using it so use it!

StringUtils.isNotBlank(groupString)